packages/markdown/components/Image/index.jsx

```diff
@@ -1,18 +1,52 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 /* eslint-disable react/jsx-props-no-spreading */
 
 const React = require('react');
 const PropTypes = require('prop-types');
 
-const Image = props => {
-  const { align, title, alt, width, height } = props;
-  const extras = { align, width, height };
-  return <img {...props} alt={alt} title={title} {...extras} loading="lazy" />;
-};
+class Image extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { lightbox: false };
+  }
+
+  render() {
+    const { align, title, alt, width, height, className = '' } = this.props;
+    const extras = { align, width, height };
+    return (
+      <React.Fragment>
+        <img
+          {...this.props}
+          alt={alt}
+          title={title}
+          {...extras}
+          loading="lazy"
+          onClick={() => className !== 'emoji' && this.setState({ lightbox: true })}
+        />
+        {className === 'emoji' || (
+          <span
+            className="lightbox"
+            onClick={() => this.setState({ lightbox: false })}
+            onScrollCapture={() => this.setState({ lightbox: false })}
+            open={this.state.lightbox}
+            role="dialog"
+          >
+            <span className="lightbox-inner">
+              <img {...this.props} alt={alt} title="Click to close..." {...extras} loading="lazy" />
+            </span>
+          </span>
+        )}
```

Comment:
This is a very simple, markup and CSS-driven lightbox. My goal here was mainly to keep the necessary Javascript as quick, simple, and lightweight (this requires zero additional deps!) as possible.